dashboard/app/controllers/level_sources_controller.rb

```diff
@@ -100,6 +100,7 @@ def set_level_source
     else
       @level_source = LevelSource.where(hidden: false).find(params[:id])
     end
+    @level_source.try(:replace_old_when_run_blocks)
```

Comment:
the .try isn't really doing much here because it will crash on the very next line if @level_source is nil...